dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -197,7 +197,17 @@ public Connection getConnection() throws SQLException {
         return hs.toArray(array);
     }
 
-    protected abstract Pattern getParameterPattern();
+    protected String defaultParamPatternString = "";
+
+    protected String getParamPatternString() {
```

Comment:
Can we keep that method private (if we want a separate method at all)?

Reply:
Ah, I see we're using it in subclasses. I'm wondering about the `getParameterPattern().toString()` alternative. My point is that having too many methods which is possible to override might lead to some confusion when implementing a new adapter.